### PR TITLE
improve documentation

### PR DESCRIPTION
**Pull request by slhck**
- Create an index page for all documentation pages
- Clean up README.md to reference documentation overview, from where people can find all docs
- Harmonize Markdown headings in all individual pages
- Add dedicated page for core metrics
- Add NEG to model page
- Harmonize naming of Markdown documents
- Move PDFs to separate directories
- Fix a few spelling errors, broken HTTP(S) links etc.

This somewhat fixes https://github.com/Netflix/vmaf/issues/125

### CODE REVIEW

File: resource/doc/metrics.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+### Motion2
+
+This is a simple measure of the temporal difference between adjacent frames. This is accomplished by calculating the average absolute pixel difference for the luminance component.
+
+### ADM
+
+ADM was previously named Detail Loss Metric (DLM), described in S. Li, F. Zhang, L. Ma, and K. Ngan, “Image Quality Assessment by Separately Evaluating Detail Losses and Additive Impairments,” IEEE Transactions on Multimedia, vol. 13, no. 5, pp. 935–949, Oct. 2011.
+
+DLM is an image quality metric based on the rationale of separately measuring the loss of details which affects the content visibility, and the redundant impairment which distracts viewer attention. The original metric combines both DLM and additive impairment measure (AIM) to yield a final score. In VMAF, only the DLM part is added as an elementary metric. Particular care was taken for special cases, such as black frames, where numerical calculations for the original formulation break down.
```
Comment (slhck) [inline resource/doc/metrics.md:21]: I wasn't too sure about this one, and I'm missing some details here (no pun intended). Please add accordingly.

PS: I took the liberty of quoting from the blog post here. If this is an issue, I can rephrase.

Comment (slhck) [reply resource/doc/metrics.md:21]: Well it's a verbatim copy from the Netflix blog post. Hence my PS — but if it's fine for you, it's fine for me!

Comment (MartinHawes10) [reply resource/doc/metrics.md:21]: What does ADM stand for?

Comment (li-zhi) [reply resource/doc/metrics.md:21]: ADM was a historical name for DLM and AIM combined.

Comment (MartinHawes10) [reply resource/doc/metrics.md:21]: So ADM =  AIM DLM metric?

Comment (li-zhi) [reply resource/doc/metrics.md:21]: Yes. 

We probably should clean up the codebase to avoid the confusion. Added this to my todo list.

Comment (li-zhi) [reply resource/doc/metrics.md:21]: Fair point. Let me think about this further.

**Pull request merged**